nim_status/migrations/accounts/0001_accounts.up.sql

```diff
@@ -1,6 +1,7 @@
 CREATE TABLE IF NOT EXISTS accounts (
 keyUid VARCHAR PRIMARY KEY,
 name TEXT NOT NULL,
+creationTimestamp BIG INT,
```

Comment:
Just as a general comment for this as it is something that we should revisit in the future. Instead of making schema updates in existing migrations, it may better to create a single schema sql file that we can bring up the entire DB with. Any changes to the schema can be added to the schema definition directly. Once nim-status is out in the wild, we can update existing DBs using migrations.